### DISCUSSION TITLE
Interceptors/WebGraphQlInterceptor  in graphql kickstart  12.0.0 ?

### DISCUSSION THREAD

**Discussion by tavomtorres** [graphql-java-kickstart/graphql-spring-boot#805] [Q&A] [answered]
Hello everyone, 

I wanted to know if in version of graphql kickstart 12.0.0, does not handle WebGraphQlInterceptor ?

It is in this official documentation and it seemed useful for me.
https://docs.spring.io/spring-graphql/docs/current-SNAPSHOT/reference/html/#server-interception

I wanted to use it for each time a graphql request is made in my backend to save the user that executes it at what time and its action.

But I see that I can't implement WebGraphQlInterceptor ... is there something similar that I can use for graphql kickstart 12? My application is with WebFlux too.

Thank you in advance for your response

**Comment by oliemansm** [comment, marked as answer]
That's spring-graphql specific and is unrelated to Kickstart. You'll have to use the spring-graphql stack and not Kickstart if you want to use that approach.
